src/index.js

```diff
@@ -123,6 +126,39 @@ exports.item_to_pwn = (item) => {
   }
 }
 
+function fisherYatesShuffle(array) {
+  var currentIndex = array.length, temporaryValue, randomIndex;
+
+  // While there remain elements to shuffle...
+  while (0 !== currentIndex) {
+
+    // Pick a remaining element...
+    randomIndex = Math.floor(Math.random() * currentIndex);
+    currentIndex -= 1;
+
+    // And swap it with the current element.
+    temporaryValue = array[currentIndex];
+    array[currentIndex] = array[randomIndex];
+    array[randomIndex] = temporaryValue;
+  }
+
+  return array;
+}
+
+function usingAnonymousPwnees() {
+  return process.env.FRIENDSHIP_IS_MAGIC != null && process.env.FRIENDSHIP_IS_MAGIC == "1";
+}
+
+function getPwnedName(pwned, anonymousPwnees, index) {
+  if (anonymousPwnees ) {
+    if (index > anonymousPwnees.length-1) { //in case the pwn list ever grows out of the hardcoded 5 value and thre are more recent pwnees than the number of my little pony characters
```

Comment:
nit: `there` misspelled, and looks like the list now has 12 values. The comment does make for a long line that gets wrapped.